tests/integration/cli/classification/test_classification.py

```diff
@@ -100,7 +100,7 @@ def tmp_dir_path():
     templates_ids = [template.model_template_id for template in templates]
 
 
-class TestToolsMPAClassification:
+class TestToolsClassification:
```

Comment:
It would be better if the multi-class classification had a distinct name.

Reply:
Reflected.